extensions/amp-tiktok/0.1/amp-tiktok.js

```diff
@@ -108,6 +115,7 @@ export class AmpTiktok extends AMP.BaseElement {
       // If the user provides a src attribute extract the video id from the src
       const videoIdRegex = /^((.+\/)?)(\d+)\/?$/;
       this.videoId_ = src.replace(videoIdRegex, '$3');
+      this.oEmbedRequestUrl_ = this.videoId_ !== src ? src : null;
```

Comment:
FMI: It looks like comparing `videoId_` to `src` is important here to see if the replace operation had any modification to make. Why does that information impact what the `oEmbedRequestUrl_` should be?

I would also recommend flipping the ternary condition so the logic is more straightforward to follow:
```suggestion
      this.oEmbedRequestUrl_ = this.videoId_ === src ? null : src;
```

Reply:
In this case we are testing to see if the src is the full video src url or if it is just the video id number. Only if its the full URL do we set it as the oEmbed request URL. I will flip the ternary statement though and add a comment.